### ISSUE TITLE
[bug] Exposure behaves very strange

### ISSUE THREAD

**Issue by salvadhor** [Polarrco/polarr_linux#1]
Moving Exposure slider makes picture over-saturated and grainy. 

Can observe this on Ubuntu, Linux Mint and others.

**Comment by bwang29** [comment]
Does this happen on the web browser version too at https://v4.polarr.co ?

**Comment by salvadhor** [comment]
Yes, in web browser I can see the same results (on Linux - on Windows everything is ok).

**Comment by bwang29** [comment]
We suspect is likely to be a shader issue with OpenGL layer on your Linux GPU, and we can't do much for that as we used the same shaders for all of our platforms with certain troubles for Mali series or Adreno 3XX series GPUs on Android as well.
